src/org/infinity/datatype/ResourceRef.java

```diff
@@ -377,6 +395,10 @@ private boolean isEditable(ResourceRefEntry ref) {
     return ref != null && ref != NONE && ref.entry != null;
   }
 
+  private boolean isSound(ResourceRefEntry ref) {
+    return ref != null && ref != NONE && ref.entry != null && ref.entry.getExtension().equalsIgnoreCase("WAV");
+  }
+
```

Comment:
Probably, it is better to add a dedicated method `isSound()` to `entry`. I think, that not only `WAV` files can be played, so it is better to hide this check behind the method

Reply:
updated in https://github.com/NearInfinityBrowser/NearInfinity/pull/164/commits/ffc0d4bc2ae7811e8136c2a3b1d083d7b24a375e